ui/test/unit/clinical/patientcontrolpanel/directives/patientControlPanel.spec.js

```diff
@@ -254,4 +254,66 @@ describe('patientControlPanelTest', function () {
 
         expect(compiledElementScope.isInEditEncounterMode()).toBeTruthy();
     })
+
+    it("goToDashBoard() should return the dashboardUrl with given program stateParams",function(){
```

Comment:
@maharjunm, Indent the line. Also change **goToDashBoard()** to **goToDashBoard**

Reply:
Done. Have a look at latest commit.